ksql-engine/src/main/java/io/confluent/ksql/codegen/CodeGenRunner.java

```diff
@@ -56,7 +64,7 @@ public CodeGenRunner(Schema schema, FunctionRegistry functionRegistry) {
     this.schema = schema;
   }
 
-  public Map<String, Class> getParameterInfo(final Expression expression) {
+  public Map<String, KsqlFunction> getParameterInfo(final Expression expression) {
```

Comment:
Why changing this to `KsqlFunction`? This map is to represent the argument name and it's type.

Reply:
it is because we need to have a `Supplier` for functions rather than the class. This is an intermediate step until i get to the next stage